src/patternfly/components/FormControl/examples/form-control-input-example.hbs

```diff
@@ -1,12 +1,17 @@
-{{#> form-control controlType="input" input="true" form-control--attribute='disabled type="text" id="textInput1" aria-label="disabled text input"'}}
+{{#> form-control controlType="input" input="true" form-control--attribute='disabled type="text" value="disabled" id="textInput1" aria-label="disabled input example"'}}
 {{/form-control}}
-{{#> form-control controlType="input" input="true" form-control--attribute='type="text" id="textInput2" aria-label="standard input example"'}}
+<br><br>
+{{#> form-control controlType="input" input="true" form-control--attribute='type="text" value="standard" id="textInput2" aria-label="standard input example"'}}
 {{/form-control}}
-{{#> form-control controlType="input" input="true" form-control--attribute='readonly type="text" id="textInput3" aria-label="readonly input example" value="readonly input example"'}}
+<br><br>
+{{#> form-control controlType="input" input="true" form-control--attribute='readonly type="text" value="readonly" id="textInput3" aria-label="readonly input example"'}}
 {{/form-control}}
-{{#> form-control controlType="input" input="true" form-control--attribute='required type="text" id="textInput4" aria-invalid="true" aria-label="Error state input example"'}}
+<br><br>
+{{#> form-control controlType="input" input="true" form-control--attribute='required type="text" value="error" id="textInput4" aria-invalid="true" aria-label="Error state input example"'}}
 {{/form-control}}
+<br><br>
 {{#> form-control controlType="input" input="true" form-control--modifier="pf-m-form-control-alt" form-control--attribute=' type="text" id="textInput5" aria-label="Alternative input example"'}}
```

Comment:
I might be wrong but I thought we were ditching the alternative input style now and just having the single style. @mceledonia can confirm :)

Reply:
Confirmed! Just going with the single new input style, it's kind of a combination of the old 2 styles as well.

Reply:
thanks fellas!